docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -135,3 +135,12 @@ in Spark WebUI below.
 
 ![locality]({{site.data.img.screenshot_datalocality_tasklocality}})
 
+### Running Spark on YARN
+
+When a Spark job is run on YARN, Spark launches its executors without taking data locality into account.
+Spark will then correctly take data locality into account when deciding how to distribute tasks to its
+executors. For example, if `host1` contains `blockA` and a job using `blockA` is launched on the YARN
+cluster with `--num-executors=1`, Spark might place the only executor on `host2` and have poor locality.
+However, if `--num-executors=2` and executors are started on `host1` and `host2`, Spark will be smart
+enough to prioritize placing the job on `host1`. To maximize the amount of locality your Spark jobs attain,
```

Comment:
can we move the last statement "To maximize the amount of locality ..." up front as the recommendation for people using Spark on YARN. Otherwise it is kinda hidden in the last of the paragraph and requires people to read the entire section to dig it out.